docs/learning/solutions/index.md

```diff
@@ -1,3 +1,52 @@
-# Solutions
+---
+home: true
+title: Runbook Automation Solutions
+heroText:  Automation Solutions for Ops
+tagline: See how Automation can transform your operations and streamline processes.
+//heroImage: /assets/img/pd-background.png
+bgImage: /assets/img/pd-background.png
+bgImageDark: /assets/img/pd-background.png
+//bgImageStyle:
+//background-attachment: fixed
 
-[Learn more about Automated Diagnostics](/learning/solutions/automated-diagnostics/index.md)
\ No newline at end of file
+actions:
+- text: Getting Started
+  icon: circle-play
+  link: /learning/solutions/getting-started.md
+- text: Start a Trial
+  icon: /assets/img/pd-icon-small.png
+  link: https://www.pagerduty.com/sign-up/runbook-automation/
+- text: Get the Code
+  icon: code-branch
+  link: https://github.com/rundeckpro/automated-diagnostics-project
+- text: Learning Rundeck
+  icon: graduation-cap
+  link: /learning/index.md
+
+highlights:
+
+- header: Solution Categories
+  description: Explore Runbook Automation solution categories
+  //image: /assets/image/markdown.svg
+  //bgImage: /assets/img/pd-background.png
+  //bgImageDark: /assets/img/pd-background.png
+  bgImageStyle:
+  background-repeat: repeat
+  background-size: initial
+  features:
+    - title: Automated Diagnostics
+      icon: book 
+      details: Gone are the days of manual troubleshooting! Empower your teams to orchestrate your container environments effortlessly, ensuring seamless integration between your infrastructure and business-critical applications. When things go sideways, use these solutions to diagnose and resolve problems before they escalate.
```

Comment:
"orchestrate your container environments..." Are we intentionally focusing on containers here?

Reply:
It was text from Joey. No, we don't have to